simulation-app/database/init.sql

```diff
@@ -0,0 +1,13 @@
+CREATE SEQUENCE user_id_seq;
+CREATE TABLE IF NOT EXISTS output_files (
+    id SERIAL NOT NULL,
+    run_file BYTEA,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS input_files (
+    name TEXT NOT NULL PRIMARY KEY,
+    content BYTEA
+);
+
+
```

Comment:
This is good for this task, but I imagine this will probably need to be done using a scripting language in future iterations since tables columns will be dynamic.